content/en/container-image.md

```diff
@@ -4,8 +4,14 @@ status: Feedback Appreciated
 category: concept
 ---
 
+## What it is
+
 A container image is an immutable, static file containing the dependencies for the creation of a container. These dependencies may include a single executable binary file, system libraries, system tools, environment variables, and other required platform settings. Container images result from an application's containerization and are typically stored in container registries, where they can be downloaded and run as an isolated process using a Container Runtime Interface (CRI). A container image framework must follow the standard schema defined by the Open Container Initiative (OCI).
 
+## Problem it addresses 
+
 Traditionally, application servers are configured per environment, and applications are deployed to them. Any misconfiguration between environments is problematic and often leads to downtime or failed deployments. An application's environment needs to be repeatable and well-defined; otherwise, the chance of environment-related bugs increases. When application environments are underdefined or inaccurate, horizontal and vertical scaling of applications becomes challenging. 
```

Comment:
Oops, I think there is a typo here. It might be `undefined` rather than `underdefined`.

I won't change it for now until you approve :)